app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::V1::ReservationsController < ApplicationController
+  def index
+    @reservations = current_user.reservations.to_json(include: :vehicle)
+    render json: @reservations, status: 200
+    # @kenny
+  end
+
+  def create
+    # @vehicle = Vehicle.find(params[:id])
+    # @user = User.find(params[:id])
+    @reservation = current_user.reservations.new(reservation_params)
+    # @reservation = Reservation.new(reservation_params)
+    # @reservation.user_id = @user.id
+    # @reservation.vehicle_id = @vehicle.id
```

Comment:
- [ ] Kindly, remove this old commented code in order to preserve your repo's cleanliness. 👍